notebooks/metagenomics_samplesheet_generator.ipynb

```diff
@@ -829,9 +904,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[31mErrorMessage: \u001b[0mIn the Bioinformatics section Project #1 does not have experiment_design_description specified\n"
```

Comment:
Can you add a sample experiment design description in this cell?

Reply:
@ElDeveloper sure I can take a look in the morning